plugins/package-managers/gradle-inspector/src/main/kotlin/GradleDependencyHandler.kt

```diff
@@ -76,7 +73,7 @@ internal class GradleDependencyHandler(
         val id = identifierFor(dependency)
         val model = dependency.mavenModel ?: run {
             issues += createAndLogIssue(
-                source = issueSource,
+                source = GradleInspectorFactory.descriptor.displayName,
```

Comment:
Why isn't the same also done for `MavenDependencyHandler`?

Reply:
Because it's used from both `Maven` and `Tycho`, so I could not hardcode it.